rmgpy/data/kinetics/groups.py

```diff
@@ -218,15 +220,42 @@ class together, returning their product as a new kinetics object of
             )
```

Comment:
Maybe the entire __multiplyKineticsData method should be moved into the KineticsData class (or at least moved outside the KineticsGroups class), since it doesn't use any methods from that class. Any thoughts on this?